hil/cli.py

```diff
@@ -288,570 +194,194 @@ def serve_networks():
-        if len(args) == 4:
-            switchinfo = {"type": switch_api + subtype, "hostname": args[0],
-                          "username": args[1], "password": args[2],
-                          "interface_type": args[3]}
-        else:
-            sys.stderr.write('ERROR: subtype ' + subtype +
-                             ' requires exactly 4 arguments\n'
-                             '<hostname> <username> <password> '
-                             '<interface_type>\n'
-                             'NOTE: interface_type refers '
-                             'to the speed of the switchports\n '
-                             'ex. TenGigabitEthernet, FortyGigabitEthernet, '
-                             'etc.\n')
-            return
-    else:
-        sys.stderr.write('ERROR: Invalid subtype supplied\n')
-        return
-    url = object_url('switch', switch)
-    do_put(url, data=switchinfo)
-
-
-@cmd
-def switch_delete(switch):
-    """Delete a <switch> """
-    C.switch.delete(switch)
-
-
-@cmd
-def list_switches():
-    """List all switches"""
-    q = C.switch.list()
-    sys.stdout.write('%s switches :    ' % len(q) + " ".join(q) + '\n')
-
-
-@cmd
-def port_register(switch, port):
-    """Register a <port> with <switch> """
-    C.port.register(switch, port)
-
-
-@cmd
-def port_delete(switch, port):
-    """Delete a <port> from a <switch>"""
-    C.port.delete(switch, port)
-
-
-@cmd
-def port_connect_nic(switch, port, node, nic):
-    """Connect a <port> on a <switch> to a <nic> on a <node>"""
-    C.port.connect_nic(switch, port, node, nic)
-
-
-@cmd
-def port_detach_nic(switch, port):
-    """Detach a <port> on a <switch> from whatever's connected to it"""
-    C.port.detach_nic(switch, port)
-
-
-@cmd
-def port_revert(switch, port):
-    """Detach a <port> on a <switch> from all attached networks."""
-    print C.port.port_revert(switch, port)
-
-
-@cmd
-def list_network_attachments(network, project):
-    """List nodes connected to a network
-    <project> may be either "all" or a specific project name.
-    """
-    print C.network.list_network_attachments(network, project)
-
-
-@cmd
-def list_nodes(is_free):
-    """List all nodes or all free nodes
-
-    <is_free> may be either "all" or "free", and determines whether
-        to list all nodes or all free nodes.
-    """
-    q = C.node.list(is_free)
-    if is_free == 'all':
-        sys.stdout.write('All nodes %s\t:    %s\n' % (len(q), " ".join(q)))
-    elif is_free == 'free':
-        sys.stdout.write('Free nodes %s\t:   %s\n' % (len(q), " ".join(q)))
-    else:
-        sys.stdout.write('Error: %s is an invalid argument\n' % (is_free))
-
-
-@cmd
-def list_project_nodes(project):
-    """List all nodes attached to a <project>"""
-    q = C.project.nodes_in(project)
-    sys.stdout.write('Nodes allocated to %s:  ' % project + " ".join(q) + '\n')
-
+###########################
+# I am doing the dash thing with the console commands. Let me know what's
```

Comment:
+1 from me. Unless a better way of handling this. We should use a `-` to connect a two-word action.

Reply:
see how I did nic.